docs/agent/upgrade.md

```diff
@@ -1,12 +1,3 @@
 # Upgrade to Agent 6
 
-Datadog Agent 6 has officially been released. Please use the in-application [install instructions](https://app.datadoghq.com/account/settings#agent) to install Agent 6 from scratch or upgrade from Agent 5.
-
-For a list of changes between Agent 5 and 6 that may affect you, please refer to:
-* [the doc on the general Agent changes][changes]
-* [the doc on the differences in the Agent config options][config]
-* [the list of Agent 5 features that are not ported to Agent 6][missing-features]
-
-[changes]: changes.md
-[config]: config.md
-[missing-features]: missing_features.md
+Instructions to upgrade your Agent from version 5 to version 6 can be found in the [Datadog public documentation](https://docs.datadoghq.com/agent/guide/upgrade-to-agent-v6/).
```

Comment:
Should we use the `#upgrade-to-agent-6` anchor https://docs.datadoghq.com/agent/guide/upgrade-to-agent-v6/#upgrade-to-agent-6 ?

Reply:
No need i think, If people do reach this page through the datadog-agent repo, having some deductive approach where Agent v6 is explained before instructions on how to install/upgrade it are given can't hurt.

Reply:
ok